Cargo.toml

```diff
@@ -83,6 +84,12 @@ path = "src/schema_generator.rs"
 test = false
 bench = false
 
+[[bin]]
+name = "wal_inspector"
+path = "src/wal_inspector.rs"
```

Comment:
Should we maybe move this to the [`src/bin/`](https://doc.rust-lang.org/cargo/guide/project-layout.html) directory?

I believe then we can even omit this `Cargo.toml` change.

Reply:
That's a good point, I propose to move both binaries in a follow up PR to keep things clean :+1: